processor/filterprocessor/config.go

```diff
@@ -68,6 +85,16 @@ type SpanFilters struct {
 	// all other spans should be included.
 	// If both Include and Exclude are specified, Include filtering occurs first.
 	Exclude *filterconfig.MatchProperties `mapstructure:"exclude"`
+
+	// Span is an OTTL condition for an ottlspan context.
+	// If the condition resolves to true, the span will be dropped.
+	// Supports `and`, `or`, and `()`
+	Span string `mapstructure:"span"`
+
+	// SpanEvent is an OTTL condition for an ottlspanevent context.
+	// If the condition resolves to true, the span event will be dropped.
+	// Supports `and`, `or`, and `()`
+	SpanEvent string `mapstructure:"spanevent"`
```

Comment:
Should these be under a "traces" entry (not spans) to be consistent with logs/metrics which are the name of the signals.

Reply:
>I could create the traces section now and only include the new config options, or we could do a breaking change at a later time.

Yes only the new config should be there I think, the old one we are planning to deprecate anyway.

Reply:
Done